src/domain/prices.ts

```diff
@@ -56,7 +67,25 @@ function fillGaps(prices, periodSeconds) {
   return newPrices;
 }
 
-async function getChartPricesFromStats(chainId, symbol, period) {
+export async function getLimitChartPricesFromStats(chainId, symbol, period, limit = 1) {
+  if (["WBTC", "WETH", "WAVAX"].includes(symbol)) {
```

Comment:
code to normalize symbol name is duplicated, let's move it to separate function?